### PR TITLE
Warnings

### PR DESCRIPTION
**Pull request by StasJ**

### CODE REVIEW

File: include/vapor/VolumeRenderer.h [hunk 1]
```diff
@@ -58,10 +58,10 @@ class RENDER_API VolumeRenderer : public Renderer
...
     bool _needToSetDefaultAlgorithm() const;
     
-    unsigned int _VAO = NULL;
-    unsigned int _VBO = NULL;
-    unsigned int _VAOChunked = NULL;
-    unsigned int _VBOChunked = NULL;
+    unsigned int _VAO = (int)NULL;
+    unsigned int _VBO = (int)NULL;
+    unsigned int _VAOChunked = (int)NULL;
+    unsigned int _VBOChunked = (int)NULL;
```
Comment (shaomeng) [inline include/vapor/VolumeRenderer.h]: 1. I think we better give these `unsigned int` value of 0?

File: include/vapor/VolumeRenderer.h [hunk 2]
```diff
@@ -58,10 +58,10 @@ class RENDER_API VolumeRenderer : public Renderer
...
     VolumeAlgorithm *_algorithm = NULL;
```
Comment (shaomeng) [inline include/vapor/VolumeRenderer.h]: We decided to adtop c++11 convention, so NULL is discouraged, and nullptr is encouraged.

File: lib/render/ShaderManager.cpp [hunk 3]
```diff
@@ -106,7 +106,7 @@ SmartShaderProgram ShaderManager::GetSmartShader(const std::string &name)
 int ShaderManager::LoadResourceByKey(const std::string &key)
 {
 //    printf("Begin Compile %s\n", key.c_str());
-    void *t = GLManager::BeginTimer();
+//    void *t = GLManager::BeginTimer();
```
Comment (shaomeng) [inline lib/render/ShaderManager.cpp]: Remove commented code? The same applies to a few other comments.

Comment (StasJ) [reply lib/render/ShaderManager.cpp]: Those are there for debugging in the future

File: include/vapor/VolumeRenderer.h (hunk 1 above)
Comment (StasJ) [reply include/vapor/VolumeRenderer.h]: These are handles that happen to be represented as integers. NULL indicates that they are not just numbers.

File: include/vapor/VolumeRenderer.h (hunk 2 above)
Comment (StasJ) [reply include/vapor/VolumeRenderer.h]: Fixed in #b2e2b92

**Pull request merged**